sharepoint/sharepoint-ps/sharepoint-online/Get-SPOApplication.md

```diff
@@ -32,7 +32,7 @@ Get-SPOApplication [[-OwningApplicationId] <OwningApplicationid>] [[-Application
 
 ## DESCRIPTION
 
-The `Get-SPOApplication` cmdlet retrieves and returns all third-party SharePoint Embedded applications registered in a tenant that match the given criteria. You must be a SharePoint Online Administrator or Global Administrator to run the cmdlet. For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps). 
+The `Get-SPOApplication` cmdlet retrieves and returns SharePoint Embedded applications of all publishers registered in a tenant that match the given criteria. You must be a SharePoint Online Administrator or Global Administrator to run the cmdlet. For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps). 
```

Comment:
>or Global Administrator [](http://example.com/codeflow?start=205&length=23)

"Global Administrator" is being de-emphasized in our PowerShell docs to encourage folks to use lesser privilege roles, so please remove all references to "Global Administrator".

The Docs team has already done 1 pass to remove this from existing docs